api/RTCDataChannel.json

```diff
@@ -430,10 +430,10 @@
               "version_added": "79"
             },
             "firefox": {
-              "version_added": false
+              "version_added": true
```

Comment:
Out of curiosity, I tested this and it works as far back as version 17 (which clashes with the root feature's 18 for DataChannel). Same goes for `onerror`. Not sure whether you want to fight with that oddity in this PR or not.

Reply:
If possible, as there a lot of `true` already for Firefox,  I would like not to fight it now. There is likely a lot of oddities for Firefox. (And I'm only cleaning a bit the content to kill a macro)